Add a markdown image preview for the node help menu.

comfyui-wiki opened the pull request:
When updating the node docs, I found sometimes it's hard to preview the image, so I made this PR.
The code was written by Claude. I'm not good at development.


https://github.com/user-attachments/assets/667c8922-f4cc-49b2-99ab-6fe4aa2a9987


┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-4417-Add-a-markdown-image-preview-for-the-node-help-menu-22c6d73d3650818abf67d9f563525af2) by [Unito](https://www.unito.io)

comfyui-wiki committed:
Add a markdown image preview for the node help menu.

christian-byrne reviewed (commented):
Can we move it into a new component that wraps ResultGallery? We also most likely don't need to use ResultGallery and can just use Galleria

comfyui-wiki committed:
refactor: extract markdown image gallery to separate component

- Create new MarkdownImageGallery component using PrimeVue Galleria directly
- Remove dependency on ResultGallery and ResultItemImpl
- Simplify image data structure with simple ImageItem interface
- Improve separation of concerns and maintainability
- Address review feedback to use dedicated component wrapper

comfyui-wiki committed:
Remove extra line breaks and clean up the code

christian-byrne commented:
@benceruleanlu Could you handle doing all reviews for node help menu features/fixes?

benceruleanlu commented:
@claude I remember we have similar functionality for the output sidebar tab, where users can click on images to expand fully, similar to what we have here.

Do you think we can reuse the code here as well?

christian-byrne commented:
> Should I add myself to CODEOWNERS?

I added you to https://github.com/orgs/Comfy-Org/teams/comfy_frontend_devs

comfyui-wiki commented:
Feel free to make any changes. Overall, the whole code is completed by CC.

benceruleanlu reviewed (dismissed):
LGTM

followup issue: we can refactor this and merge it with ResultGallery.vue to turn it into a common component, using a type adapter

benceruleanlu commented:
second followup issue: adding e2e browser tests to make sure this works well

actions-user committed:
[automated] Apply ESLint and Prettier fixes
